### PR TITLE
Implemented Liberator absorb spell bonuses and added Liberator Lv 75~…

### PR DESCRIPTION
**Pull request by IxAalexI**
Implemented Liberator absorb spell bonuses and added Liberator Lv 75~99-4 to item_mods.sql.

### CODE REVIEW

File: scripts/globals/status.lua [hunk 1]
```diff
@@ -1230,11 +1230,14 @@ MOD_SHARPSHOT       = 0x13A -- Sharpshot accuracy bonus (modId = 314)
...
 
+-- Mythic Weapon Mods
+MOD_LIBERATOR_ABSORB_BONUS    = 0x209  -- Direct Absorb spell increase while Liberator is equipped (percentage based) (modId = 521)
+
 -- The entire mod list is in desperate need of kind of some organizing.
 -- The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
 
--- MOD_SPARE = 0x209 -- (modId = 521)
--- MOD_SPARE = 0x20A -- (modId = 522)
+-- MOD_SPARE = 0x20A, // (modId = 522)
```
Comment (TeoTwawki) [inline scripts/globals/status.lua]: lua syntax error. Copy paste fail? I do that here myself some times. Do a git amend, take out the commas and exchange the slashes for dashes.

File: sql/item_mods.sql [hunk 2]
```diff
@@ -21567,9 +21562,6 @@ INSERT INTO `item_mods` VALUES (19716,27,16);
 INSERT INTO `item_mods` VALUES (19716,161,-16);
 INSERT INTO `item_mods` VALUES (19716,288,25);
 INSERT INTO `item_mods` VALUES (19716,302,5);
-INSERT INTO `item_mods` VALUES (19717,30,30);
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: DO NOT DO THAT

Comment (TeoTwawki) [inline sql/item_mods.sql]: If you have new liens for the same item, add them here. Don't go trying to sort the list by groups of "types" of item, like jamming all mythic in one spot.

File: src/map/modifier.h [hunk 3]
```diff
@@ -568,11 +568,12 @@ enum MODIFIER
     MOD_SHIELDBLOCKRATE           = 0x206, // Affects shield block rate, percent based (modID = 518)
     MOD_DIA_DOT                   = 0x139, //Increases the DoT damage of Dia (modId = 313)
     MOD_ENH_DRAIN_ASPIR           = 0x13B, // % damage boost to Drain and Aspir(modId = 315)
-    MOD_TRICK_ATK_AGI             = 0x208 // % AGI boost to Trick Attack (if gear mod, needs to be equipped on hit) (modId = 520)
-
-    // MOD_SPARE = 0x209, // (modId = 521)
+    MOD_TRICK_ATK_AGI             = 0x208, // % AGI boost to Trick Attack (if gear mod, needs to be equipped on hit) (modId = 520)
+	
+	MOD_LIBERATOR_ABSORB_BONUS    = 0x209  // Direct Absorb spell increase while Liberator is equipped (percentage based) (modId = 521)
```
Comment (TeoTwawki) [inline src/map/modifier.h]: Hi there extra new line and a tab where preceeding liens used 4 spaces.

File: src/map/modifier.h [hunk 4]
```diff
@@ -568,11 +568,12 @@ enum MODIFIER
...
-    MOD_TRICK_ATK_AGI             = 0x208 // % AGI boost to Trick Attack (if gear mod, needs to be equipped on hit) (modId = 520)
-
-    // MOD_SPARE = 0x209, // (modId = 521)
+    MOD_TRICK_ATK_AGI             = 0x208, // % AGI boost to Trick Attack (if gear mod, needs to be equipped on hit) (modId = 520)
+	
+	MOD_LIBERATOR_ABSORB_BONUS    = 0x209  // Direct Absorb spell increase while Liberator is equipped (percentage based) (modId = 521)
+	
     // MOD_SPARE = 0x20A, // (modId = 522)
-
+	// MOD_SPARE = 0x20B, // (modId = 523)
```
Comment (TeoTwawki) [inline src/map/modifier.h]: here as well: use 4 spaces

File: sql/item_mods.sql (hunk 2 above)
Comment (IxAalexI) [inline sql/item_mods.sql]: That what :x:

Comment (TeoTwawki) [inline sql/item_mods.sql]: The only time a line in item_mods.sql should be deleted is if the line is simply wrong or obsoleted by a refactor (like pet mods were when they moved to their own table)

Comment (IxAalexI) [inline sql/item_mods.sql]: I can go back and re-insert them sequentially

File: src/map/modifier.h (hunk 4 above)
Comment (IxAalexI) [inline src/map/modifier.h]: Will do

File: scripts/globals/status.lua (hunk 1 above)
Comment (IxAalexI) [inline scripts/globals/status.lua]: I can change that np, but isn't the whole line commented out by the double dashes at the line start?

File: sql/item_mods.sql (hunk 2 above)
Comment (TeoTwawki) [inline sql/item_mods.sql]: Yes please, keep the format/sort intact. I've found it helps avoid certain types of user error when multiple editors go at this file.

**Comment by TeoTwawki** [comment]
You don't have to close pull requests to update them. Anytime you push to that branch the pull request changes contents to reflect whats in the branch.

File: sql/item_mods.sql (hunk 2 above)
Comment (IxAalexI) [inline sql/item_mods.sql]: Ok, pretty sure I tracked everything down and corrected. Closed this pull request and opened another

**Comment by IxAalexI** [comment]
Oh, oops. Well it looks cleaner in the new pull request at least.

**Pull request closed without merging**